apps/pie-storybook/stories/pie-chip.stories.ts

```diff
@@ -110,7 +111,8 @@ const Template: TemplateFunction<ChipProps> = ({
                 ?isLoading="${isLoading}"
                 ?isDismissible="${isDismissible}"
                 variant="${ifDefined(variant)}"
-                @pie-chip-close="${closeAction}">
+                @pie-chip-close="${closeAction}"
+                @click="${clickAction}">
```

Comment:
is this misleading, as there is no `pie-chip-click` event? Should it just be `click`?

Reply:
if i am not mistaken, you need to prefix native click events with @ in lit 👀 unless I missed your point.

Reply:
I mean the string value that the `clickAction` logs :)

Reply:
oh makes sense, updated